app-text/diffpdf/files/diffpdf-2.1.3-qt6.patch

```diff
@@ -0,0 +1,401 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..c7e6d76
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,25 @@
++cmake_minimum_required(VERSION 3.28)
++project(diffpdf CXX)
++
++find_package(Qt6 REQUIRED COMPONENTS LinguistTools PrintSupport Widgets)
```

Comment:
Use of QPixmap, QPixmapCache, QMimeData, QTranslator, ... suggests also Gui module being needed.

Reply:
`Gui` is currently pulled in automatically by `Widgets`, but theoretically this could change in the future, although unlikely. Will change.`